### PR TITLE
fix README for admission webhook test image

### PR DESCRIPTION
**Pull request by CaoShuFeng**
This README is copied from somewhere else and it's out of date.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ixdy** [comment]
where is this copied from? should we just link to it instead?

**Comment by CaoShuFeng** [comment]
> where is this copied from?

https://github.com/caesarxuchao/example-webhook-admission-controller/blob/master/README.md

> should we just link to it instead?

That one is deprecated now. And it only works for kubernetes 1.8.

File: test/images/webhook/README.md [hunk 1]
```diff
@@ -1,45 +1,13 @@
-# Kubernetes External Admission Webhook Example
+# Kubernetes External Admission Webhook Test Image
 
-The example shows how to build and deploy an external webhook that only admits
-pods creation and update if the container images have the "grc.io" prefix.
+The image tests MutatingAdmissionWebhook and ValidatingAdmissionWebhook. After deploying
+it to kubernetes cluster. Administrator needs to create a ValidatingWebhookConfiguration
```
Comment (caesarxuchao) [inline test/images/webhook/README.md]: After deploying it to kubernetes cluster, administrator...

**Comment by caesarxuchao** [comment]
lgtm after addressing the nits. Thanks!

File: test/images/webhook/README.md (hunk 1 above)
Comment (CaoShuFeng) [reply test/images/webhook/README.md]: Done.

**Pull request merged**